contracts/staking/cw20-stake-external-rewards/src/contract.rs

```diff
@@ -37,36 +37,29 @@ pub fn instantiate(
 ) -> Result<Response<Empty>, ContractError> {
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
 
-    let owner = msg.owner.map(|a| deps.api.addr_validate(&a)).transpose()?;
-    let manager = msg
-        .manager
-        .map(|a| deps.api.addr_validate(&a))
-        .transpose()?;
+    cw_ownable::initialize_owner(deps.storage, deps.api, msg.owner.as_deref())?;
 
     let reward_token = match msg.reward_token {
         Denom::Native(denom) => Denom::Native(denom),
         Cw20(addr) => Cw20(deps.api.addr_validate(addr.as_ref())?),
     };
 
-    if msg.reward_duration == 0 {
-        return Err(ContractError::ZeroRewardDuration {});
-    }
-
```

Comment:
We sure we want to remove this?

Reply:
moved it down to line 59 :) before we had a comment there saying we checked it above. figured it made more sense to just remove the comment and check the value where we use it.
<img width="502" alt="image" src="https://user-images.githubusercontent.com/30676292/210279564-1c83b8ba-c1b6-4815-8add-2b9cc6e760e0.png">